stripper/ze_paranoid_rezurrection_csgo1.cfg

```diff
@@ -9,14 +9,63 @@ modify:
 	}
 	insert:
 	{
+		"OnMapSpawn" "player,AddOutput,rendermode 0,,1"
+		"OnMapSpawn" "player,AddOutput,gravity 1,,1"
+		"OnMapSpawn" "player,SetDamageFilter,,,1"
+		"OnMapSpawn" "player,Color,255 255 255,,1"
 		"OnMapSpawn" "protect_player,AddOutput,solid 2,,1"
 		"OnMapSpawn" "protect_playerRunScriptCodeself.SetSize(Vector(-1600, -1178.5, -184.017), Vector(1600, 1178.5, 616));1"
+		"OnMapSpawn" "map1anticheat1,AddOutput,solid 2,,1"
+		"OnMapSpawn" "map1anticheat1RunScriptCodeself.SetSize(Vector(-0.5, -8, -302), Vector(0.5, 8, 302));1"
+		"OnMapSpawn" "protect_player,Enable,,1,1"
 		"OnMapSpawn" "playerRunScriptCodeif (self.GetTeam() == 2 && self.GetHealth() != 0) { self.SetOrigin(Vector(-12408, -8140, 318)); self.SetAngles(0, -89, 0); }301"
 	}
 	delete:
 	{
 		"OnMapSpawn" "consolaCommandmp_buytime 22-1"
 		"OnMapSpawn" "protect_playerEnable5-1"
+		"OnMapSpawn" "playerAddOutputrendermode 03-1"
+		"OnMapSpawn" "playerAddOutputgavity 13-1"
+		"OnMapSpawn" "playerSetDamageFilter3-1"
+	}
+}
+
+// Fix map giving players the "player" targetname, messing up the "player" "classname" targeted outputs.
+modify:
+{
+	match:
+	{
+		"classname" "trigger_multiple"
+		"targetname" "zm_sorpresa"
+	}
+	delete:
+	{
+		"OnStartTouch" "!activatorAddOutputtargetname player0-1"
+	}
+	insert:
+	{
+		"OnStartTouch" "!activator,AddOutput,targetname ,,-1"
+	}
+}
+
+modify:
+{
+	match:
+	{
+		"classname" "trigger_multiple"
+		"targetname" "clear_player"
+	}
+	delete:
+	{
+		"OnStartTouch" "!selfRunScriptCodeClearPlayer();0-1"
+	}
+	insert:
+	{
+		"OnStartTouch" "!activator,AddOutput,rendermode 0,,-1"
+		"OnStartTouch" "!activator,AddOutput,gravity 1,,-1"
+		"OnStartTouch" "!activator,SetDamageFilter,,,-1"
+		"OnStartTouch" "!selfRunScriptCodeforeach (A, _ in {monstruo = 0}) if (activator.GetName() != A) foreach (A, _ in {FireUser1 = 0}) EntFireByHandle(self, A, A, 0, activator, null);-1"
```

Comment:
Aren't you overriding your `A` variable here in the 2nd loop? You're effectively running `EntFireByHandle(self, "FireUser1", "FireUser1", 0, activator, null)`. Using `B` there instead would prevent that.

Reply:
This is actually fine, I got parameter order mixed up and the 2nd A is effectively ignored anyways.

Reply:
I had to input a string for the parameter string of EntFire, I couldn't figure out a way to put an empty string and 0.tostring() didn't work so I had to do that.